src/test/java/ru/training/at/hw2/ex1/ExerciseOne.java

```diff
@@ -0,0 +1,169 @@
+package ru.training.at.hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.dataproviders.DataStoreForHomeworkTwo;
+
+import static io.github.bonigarcia.wdm.config.DriverManagerType.CHROME;
+
+
+public class ExerciseOne {
+    WebDriver driver;
+    WebElement element;
+    DataStoreForHomeworkTwo dp;
+
+    @BeforeClass(groups = {"exercise_1"})
+    public void setUp() {
+        dp = new DataStoreForHomeworkTwo();
+
+        WebDriverManager.getInstance(CHROME).setup();
+        driver = new ChromeDriver();
+    }
+
+    @AfterClass(groups = {"exercise_1"})
+    public void tearDown() {
+        driver.quit();
+    }
+
+        
+    @Test(groups = {"exercise_1"})
+    public void allExerciseOneTest() {
+        openSiteByUrlTest();
+        browserTitleTest();
+        performLoginTest();
+        usernameIsLogginedTest();
+        headerHaveProperTextsTest();
+        fourImagesOnTheIndexPageAndTheyAreDisplayed();
+        fourIconsInHomePageExistsTest();
+        fourTextsOnTheIndexPageUnderIconsAndTheyHaveProperTextTest();
+        iframeWithFrameButtonExistTest();
+        thereIsFrameButtonInTheIframeTest();
+        switchToOriginalWindowBackTest();
+        thereAreFiveItemsInTheLeftSectionTest();
+    }
+
+    public void openSiteByUrlTest() {
+        driver.navigate().to(dp.getSiteUrl());
+        driver.getTitle();
+    }
+
+    public void browserTitleTest() {
+        driver.navigate().to(dp.getSiteUrl());
+        Assert.assertEquals(driver.getTitle(), dp.getBrowserTitle());
+    }
+
+    public void performLoginTest() {
+        driver.manage().window().maximize();
+
+        element = driver.findElement(By.id("user-icon"));
+        element.click();
+
+        element = driver.findElement(By.id("name"));
+        element.sendKeys(dp.getUserName());
+
+        element = driver.findElement(By.id("password"));
+        element.sendKeys(dp.getPassword());
+
+        element = driver.findElement(By.id("login-button"));
+        element.click();
+    }
+
+    public void usernameIsLogginedTest() {
+        element = driver.findElement(By.id("user-name"));
+        Assert.assertEquals(element.getText(), dp.getUserNameAfterLogged());
+    }
+
+    public void headerHaveProperTextsTest() {
+        for (int i = 1; i <= 4; i++) {
+            element = driver.findElement(By.cssSelector(
+                    "ul.uui-navigation.nav.navbar-nav.m-l8 >li:nth-child(" + i + ")"));
+            Assert.assertEquals(element.getText(),
+                    dp.getTextOfHeaderMenuButtonsList().get(i - 1));
+        }
```

Comment:
Would be better to use not hardcoded values for iteration
```suggestion
        for (int i = 0; i < getTextOfHeaderMenuButtonsList().size(); i++) {
            element = driver.findElement(By.cssSelector(
                    "ul.uui-navigation.nav.navbar-nav.m-l8 >li:nth-child(" + (i + 1) + ")"));
            Assert.assertEquals(element.getText(),
                    dp.getTextOfHeaderMenuButtonsList().get(i));
        }
```

\+ same for `fourTextsOnTheIndexPageUnderIconsAndTheyHaveProperTextTest` and `thereAreFiveItemsInTheLeftSectionTest()`